runconfig/opts/opts.go

```diff
@@ -45,6 +46,11 @@ func ValidateEnv(val string) (string, error) {
 func doesEnvExist(name string) bool {
 	for _, entry := range os.Environ() {
 		parts := strings.SplitN(entry, "=", 2)
+		if runtime.GOOS == "windows" {
+			// Environment variable are case-insensitive on Windows. PaTh, path and PATH are equivalent.
+			parts[0] = strings.ToLower(parts[0])
+			name = strings.ToLower(name)
```

Comment:
@tonistiigi Updated. But note this was the original code, not part of what I changed.